code/stats/medals.cpp

```diff
@@ -144,10 +145,9 @@ static char Medals_background_filename[NAME_LENGTH];
 static const char* Default_medals_mask_filename = "Medals-M";
 static char Medals_mask_filename[NAME_LENGTH];
 
-scoring_struct *Player_score=NULL;
-
 int Medals_mode;
 player *Medals_player;
+scoring_struct Medals_stats;
```

Comment:
`Medals_stats` should be declared `static`. Though really all 3 of these variable should be, if you're making the change anyway.